Resources/Storage/XROOTStorage.py

```diff
@@ -1597,3 +1598,32 @@ def getCurrentURL( self, fileName ):
     cwdUrl = result['Value']
     fullUrl = '%s/%s' % ( cwdUrl, fileName )
     return S_OK( fullUrl )
+
+  def constructURLFromLFN( self, lfn, withWSUrl = False ):
```

Comment:
a) I don't think it will arm either
b) Yes, calling super and adding svcClass to the result